features/data_handler_class.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+import os
+import pandas as pd
+import pandas_datareader.data as pd_dr        
+
+class DataHandlerClass():
+    
+    def __init__(self) -> None:
+        data = pd.DataFrame
+        stock = ''
+        path_to_file = ''
```

Comment:
Consider:
```python
    def __init__(self) -> None:
        self._data = pd.DataFrame
        self._stock = ''
        self._path_to_file = ''
```
'self' -> Refers to object itself, so 'self._variable' reffers to to variable assigned to the instance of class